mne/gui/_file_traits.py

```diff
@@ -285,14 +287,40 @@ def _get_n_omitted(self):
     @cached_property
     def _get_inst(self):
         if self.file:
-            info = read_info(self.file, verbose=False)
-            if info['dig'] is None:
+            try:
+                info = read_info(self.file, verbose=False)
+            except ValueError:  # try reading dig_montage
```

Comment:
I don't like silencing errors that might be real. 

Could the modification be in `read_info()`, which could create an info dict when given a montage file?